kolibri/core/hooks.py

```diff
@@ -0,0 +1,40 @@
+"""
+Kolibri Core hooks
+------------------
+
+WIP! Many applications are supposed to live inside the core namespace to make
+it explicit that they are part of the core.
+
+Do we put all their hooks in one module or should each app have its own hooks
+module?
```

Comment:
Also, I think that the navigation hook is mis-placed here. I don't wanna make `kolibri.core.hooks` a dumping site. But where to put it ATM.. hmm.